code/modules/unit_tests/unit_test.dm

```diff
@@ -14,6 +14,7 @@ You can use the run_loc_floor_bottom_left and run_loc_floor_top_right to get tur
 GLOBAL_DATUM(current_test, /datum/unit_test)
 GLOBAL_VAR_INIT(failed_any_test, FALSE)
 GLOBAL_VAR(test_log)
+GLOBAL_LIST_EMPTY(unit_test_mapping_logs)
```

Comment:
Doc-comment because this is a public API (used by mapping.dm)

Reply:
done